lebedigital/BeamDesign/BeamDesign.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 16 11:13:33 2022
+
+@author: zv1174
+"""
+import math
+
+def section_dimension_rule_of_thumb(span:float)->tuple:
+    """
+    This is as per eurocode guideline choice of section dimension based on span 
+
+    Parameters
+    ----------
+    span : float
+        beam span.
+
+    Returns
+    -------
+    tuple
+        tuple of breath and depth of the beam.
+
+    """
+    #span/depth ratio for simply supported beam is 15 Therefore,
+    d = 50*round((span/15)/50)
+    #width of beam
+    b = 0.5*d
+    return b,d 
```

Comment:
Maybe we need a general discussion on this but I don't think this is what we are looking for.
I understand that the EC might recommend a ratio of span to depth (width?) of 15, but are you required to do so?
I would prefer to directly optimize height and width, otherwise our plan of a geometry optimization is pointless.

Reply:
this function you can use for initial guess and then you can optimize further with height or width

Reply:
Now I understand. So this can be used as an initial guess.